passes/memory/memory_bram.cc

```diff
@@ -1233,6 +1309,10 @@ struct MemoryBramPass : public Pass {
 		log("has a higher efficiency than the next match (and the one after that if\n");
 		log("the next also has 'or_next_if_better' set, and so forth).\n");
 		log("\n");
+		log("A match containing the condition '!attribute' followed by a name\n");
+                log("requires that the memory does not contains the given attribute name\n");
+                log("or that the attribute is not present.\n");
```

Comment:
The two statements: "does not contains the given attribute name or that the attribute is not present." seem identical, no? Also, spacing is a little off here, and perhaps it would be better to put this description right after the `attribute` description above.